org.vinegarhq.Sober.yml

```diff
@@ -0,0 +1,71 @@
+app-id: org.vinegarhq.Sober
+runtime: org.gnome.Platform
+runtime-version: '48'
+sdk: org.gnome.Sdk
+command: sober
+separate-locales: false
+finish-args:
+  # For controller support on old Flatpak versions. Optional.
+  # SECURITY: this allows for a sandbox escape
+  # TODO: don't ship this by default. Like, really.
+  - --device=all
+  # Controller support, though this only works on newer Flatpak versions. Optional.
+  # SECURITY: shouldn't have much of an impact, although the user should disable this from Flatseal if controller support is unwanted
+  # We don't ship this by default for backwards compat :( See above.
+  # - --device=input
+  # DRI device. Required for graphics acceleration. Non-optional, since Roblox is a 3D game.
+  - --device=dri
+  # Required by the game. Non-optional for now, sadly.
+  # SECURITY: How bad is device=shm? should we try making a workaround?
+  # SECURITY: ptrace access inside the sandbox should be no big deal, really :)
+  - --device=shm
+  - --allow=devel
+  # X11 performance, also required by the game. Non-optional for now, sadly.
+  # SECURITY: How bad is sharing IPC namespaces? should we try making a workaround?
+  - --share=ipc
+  # Both Sober & Roblox require network access to work. Non-optional.
+  - --share=network
+  # Window server access is required to display the game & menus. Non-optional.
+  # SECURITY: users should try using Wayland where possible
+  - --socket=wayland
+  - --socket=fallback-x11
+  # Required for game audio & microphone access when using the optional voice chat feature. Optional, but heavily recommended.
+  # SECURITY: allows unprompted microphone access. This is a limitation of the sandbox for now.
+  - --socket=pulseaudio
+  # Required to share what Roblox experience you're playing to your Discord servers & contacts. Optional
+  # SECURITY: Sober sees your Discord username and Discord ID due to how Discord IPC works, but it shouldn't be allowed to make actions on your behalf. Also, we obviously don't send that stuff to our server.
+  - --filesystem=xdg-run/app/com.discordapp.Discord:create
+  - --filesystem=xdg-run/discord-ipc-0
+  # Required for a workaround. Has no security or UX impact, since Sober is sadly English-only for now
+  # TODO: fix the bug and remove this
+  - --env=LC_ALL=en_US.UTF-8
```

Comment:
why? Seriously. Fix your software.

Reply:
bionic isn't glibc....

en_US isn't English. It's American English. You assume the user has the en_US locale installed. This assumption is incorrect.